frontend/src/helpers/checkPassword.js

```diff
@@ -0,0 +1,20 @@
+export default (checkPassword = password => {
+  let numbers = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"];
+  let passwordArray = password.split();
+  let hasNumber = false;
+  let hasLength = false;
+  let hasUpperCase = false;
+  passwordArray.map(char => {
+    if (numbers.includes(char)) {
+      hasNumber = true;
+      return;
+    }
+    if (char.toLowerCase() < char) {
+      hasUpperCase = true;
+      return;
+    }
+  });
+  if (password.length < 7) {
+    return false;
+  }
+});
```

Comment:
I'd just let the backend worry about this validation. Having the logic duplicated in both places can cause problems down the line if they drift, and the client has different validation than the server. The client can just assume any password is valid, and forward the server error message if it's not